CriteoPublisherSdk/Tests/IntegrationTests/CR_BidManagerIntegrationTests.m

```diff
@@ -67,6 +69,9 @@ - (void)setUp {
   self.adUnit1 = [CR_TestAdUnits preprodBanner320x50];
   self.cacheAdUnit1 = [CR_AdUnitHelper cacheAdUnitForAdUnit:[CR_TestAdUnits preprodBanner320x50]];
   self.cacheAdUnit2 = [CR_AdUnitHelper cacheAdUnitForAdUnit:[CR_TestAdUnits preprodInterstitial]];
+  [Criteo.sharedCriteo.dependencyProvider.integrationRegistry declare:CR_IntegrationGamAppBidding];
```

Comment:
It might be worth adding a specific TODO with a new jira ticket (like TODO DPP-11234) so you start tracing all the places that will eventually need to be corrected when AdUnitHelper get refactored correctly.